### PR TITLE
Add missing doc tags to TimeOnly type.

### PR DESCRIPTION
**Pull request by tarekgh**

### CODE REVIEW

**Review by eerhardt — APPROVED** [review]

**Review by tannergooding — APPROVED** [review]

File: src/libraries/System.Private.CoreLib/src/System/TimeOnly.cs [hunk 1]
```diff
@@ -933,8 +933,10 @@ public bool TryFormat(Span<char> destination, out int charsWritten, [StringSynta
         // IParsable
         //
 
+        /// <inheritdoc cref="IParsable{TSelf}.Parse(string, IFormatProvider?)" />
         public static TimeOnly Parse(string s, IFormatProvider? provider) => Parse(s, provider, DateTimeStyles.None);
 
+        /// <inheritdoc cref="IParsable{TSelf}.TryParse(string?, IFormatProvider?, out TSelf)" />
```
Comment (carlossanlop) [inline src/libraries/System.Private.CoreLib/src/System/TimeOnly.cs:939]: This inheritdoc format is new to me, and the porting tool won't be able to detect it, so I will have to fix it. What is the purpose of cref here?

**Review by carlossanlop — APPROVED** [review]
LGTM but left a question.

File: src/libraries/System.Private.CoreLib/src/System/TimeOnly.cs (hunk 1 above)
Comment (tarekgh) [reply src/libraries/System.Private.CoreLib/src/System/TimeOnly.cs:939]: I don't know, I just copied and pasted from similar places. I think you can just ignore it as if it is not there if you pick the right docs from `IParsable`.

Comment (tannergooding) [reply src/libraries/System.Private.CoreLib/src/System/TimeOnly.cs:939]: @carlossanlop, the `cref` indicates that the docs come from the specified member, it allows `inheritdoc` to be used when there is no implicit mapping.

Things like summary, returns, and remarks flow automatically. Things like `param` and typeparam` only flow where the names match between the two.

**Pull request merged**